README.md

```diff
@@ -1,6 +1,8 @@
-# Sample JDL files for JHipster
+# Sample JDL files for JHipster v8
 
-This repository is a showcase of JDL schema files that are generated from JDL Studio.
+For updated JHipster v8 samples you should go to [v8 branch](https://github.com/jhipster/jdl-samples/tree/v8).
```

Comment:
@DanielFran I updated the title, but now this looks out of place. Should it point to a v7 branch?

Reply:
@mraible I believe we should remove it